### PR TITLE
Allow multiple email addresses in mutt commands

### PR DESCRIPTION
**Pull request by adamallgood**
We need the ability for keep-trying to email multiple recipients when a command fails. Here is a proposed update using a technique described [here](https://gist.github.com/narthur/52d1cd2049d4ca5af2f3)

### CODE REVIEW

File: keep-trying [hunk 1]
```diff
@@ -138,19 +139,21 @@ else
...
         fi
-        # Set sender email address
+        # Set sender email address(es)
         if [[ ! -z "$log_file" ]] ; then
             if [[ $zip_log -eq 1 ]] ; then
                 zip -rq "${log_file}.zip" $log_file
                 log_file="${log_file}.zip"
             fi
-            echo -e $email_body | mutt -s "$email_subject" $email_address -a "$log_file"
+            #echo -e $email_body | mutt -s "$email_subject" $email_address -a "$log_file"
```
Comment (mikecharles) [inline keep-trying]: Can you remove this commented out code, and the others below?

**Review by mikecharles — CHANGES_REQUESTED** [review]

File: keep-trying (hunk 1 above)
Comment (adamallgood) [reply keep-trying]: Done.

**Review by mikecharles — APPROVED** [review]
Looks good! As long as it was tested ;-)

**Comment by adamallgood** [comment]
This update is not needed. The functionality already exists.

**Pull request closed without merging**